pkgs/applications/virtualization/nvidia-docker/libnvc.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, lib, fetchFromGitHub, fetchurl, libelf, libcap, libseccomp }:
+
+with lib; let
+
+  nvidia-modprobe = fetchurl {
+    url = "https://github.com/NVIDIA/nvidia-modprobe/archive/396.51.tar.gz";
```

Comment:
You should parametrize this file by the nvidia version "396.51". It would be very easy to forget to update these numbers later down the file.